src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx

```diff
@@ -16,32 +16,211 @@ redirects:
+        * Double click an element
+        * Enter a secure credential
+        * Hover over an element
+        * Locate an element by CSS class, HTML ID, link text, Xpath, or value
+        * Navigate to a URL
+        * Select from a dropdown
+        * Type text
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Simple browser monitors
+
+        API name: `BROWSER`
+      </td>
+
+      <td>
+        Simple browser monitors essentially are simple, pre-built scripted browser monitors. They make a request to your site using an instance of Google Chrome.
+
+        Compared to a simple ping monitor, this is a more accurate emulation of an actual customer visit. The user agent is identified as `Google Chrome`.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Scripted browser monitors
+
+        API name: `SCRIPT_BROWSER`
+      </td>
+
+      <td>
+        Scripted browser monitors are used for more sophisticated, customized monitoring. You can create a custom script that navigates your website, takes specific actions, and ensures specific resources are present.
+
+        The monitor uses Google Chrome browser. You can also use a variety of [third-party modules](/docs/synthetics/new-relic-synthetics/scripting-monitors/scripted-monitor-runtime-environment#runtime-table) to build your custom monitor.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        API tests
+
+        API name: `SCRIPT_API`
+      </td>
+
+      <td>
+        API tests are used to monitor your API endpoints. This can ensure that your app server works in addition to your website. New Relic uses the [`http-request` module](https://github.com/request/request) internally to make HTTP calls to your endpoint and validate the results.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+You can [add and edit synthetic monitors](/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors) directly in our UI. You can also create some synthetic monitors with [Terraform](https://developer.newrelic.com/automate-workflows/get-started-terraform/). To learn how, watch this short YouTube video (approx. 3:25 minutes).
+
+<Video
+  id="WIVOgYlID-E"
+  type="youtube"
+/>
+
+## Advanced testing with scripted browsers [#advanced]
 
-Synthetic monitoring allows you to proactively monitor your website or API endpoint to ensure your content is not only available, but fully functional. Synthetic monitoring browser tests send real, Selenium-powered Google Chrome browsers to your site from locations around the world to ensure your content is always up, everywhere.
+Synthetic monitoring allows you to proactively monitor your website or API endpoint to ensure your content is not only available, but fully functional. [Scripted browsers](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-synthetic-scripts) send real, Selenium-powered Google Chrome instances to your site from locations around the world, so you're confident that your content is always up, everywhere.
```

Comment:
Love the "so you're confident that your content is always up, everywhere" part. I think the comma before "so you're" should be removed.